Wikipedia/Code/MWKLanguageLinkController.m

```diff
@@ -232,6 +232,16 @@ - (void)getPreferredLanguageCodes:(void (^)(NSArray<NSString *> *))completion {
     }];
 }
 
+- (void)getPreferredWikiCodes:(void (^)(NSArray<NSString *> *))completion {
```

Comment:
I'm open to an alternative name if "WikiCodes" just adds to the confusion.

Reply:
I think the naming issue is a symptom of my not trying to update existing method names to the new `langugageCode` / `languageVariantCode` / `contentLanguageCode` naming.

(I was going to suggest that for either the dot release or the clean up period that Josh mentioned that I make those changes).

So, the existing method should be `getPreferredContentLanguageCodes` and the new method should be `getPreferredLanguageCodes`.

Maybe the name `WMFPreferredLanguageInfoProviding` would work better. I think having 'language' in the name is better. We got rid of the MWKLanguageInfo type during refactoring and 'preferred language info' can contain any info about preferred languages.

Would `WMFPreferredLanguageInfoProvider` work. I always find 'Provider' to be better than 'Providing' especially since the Swift naming guidelines seem to prefer using a noun like 'Provider' if the item *is* a thing.